msteams-platform/task-modules-and-cards/cards/cards-reference.md

```diff
@@ -128,6 +128,7 @@ The following table provides the features that support Adaptive Cards:
 > * Positive or destructive action styling is not supported in Adaptive Cards on the Teams platform.
 > * Test your full width Adaptive Card in narrow form factors such as mobile and meeting side panels to ensure that content is not truncated.
 > * Adaptive Cards within Teams don't provide support for file or image uploads.
+> * The `isEnabled` property for `Action.Submit` type in an Adaptive Card isn't supported in Teams.
```

Comment:
This is not the only one property.
There may be other properties as well which is available in https://adaptivecards.io/ but actually not supported.

@surbhigupta12 - Could you please confirm whether we should add a note in the documentation that **all the available properties  in https://adaptivecards.io/ are not supported in Teams?**

Reply:
Let's go ahead with this one.